yt_dlp/extractor/rumble.py

```diff
@@ -337,14 +345,22 @@ def _real_extract(self, url):
         release_ts_str = self._search_regex(
             r'(?:Livestream begins|Streamed on):\s+<time datetime="([^"]+)',
             webpage, 'release date', fatal=False, default=None)
-        view_count_str = self._search_regex(r'<span class="media-heading-info">([\d,]+) Views',
-                                            webpage, 'view count', fatal=False, default=None)
+        like_count_str = self._search_regex(
+            r'<span data-js="rumbles_up_votes">([\d,\.KM]+)<',
+            webpage, 'like count', fatal=False, default=None)
+        dislike_count_str = self._search_regex(
+            r'<span data-js="rumbles_down_votes">([\d,\.KM]+)<',
+            webpage, 'dislike count', fatal=False, default=None)
+        view_count_str = self._search_regex(
+            r'"userInteractionCount":([\d,]+)}',
+            webpage, 'view count', fatal=False, default=None)
```

Comment:
- `fatal=False` is redundant if there is a `default`
- `.` does not need to be escaped inside a regex character class
- For `view_count`, would it be better to parse the JSON LD instead? Would save a step maybe. Or else the regex pattern should allow whitespace, e.g. `r'"userInteractionCount"\s*:\s*(\d+)'` . And it's a JSON int value, so the `,` and `parse_count` are unnecessary
- all of this might make more sense inlined in the return, and then for readability it should return an info_dict instead of `self.url_result()`, e.g.:
```py
        return {
            '_type': 'url_transparent'
            'ie_key': url_info['ie_key'],
            'url': url_info['url'],
            'like_count': parse_count(self._search_regex(
                r'<span data-js="rumbles_up_votes">\s*([\d,.KM]+)', webpage, 'like count', default=None)),
            # etc...
        }
```